src/Tools/dotnet-counters/Program.cs

```diff
@@ -54,47 +55,12 @@ internal class Program
                 new Option[] { },
                 handler: CommandHandler.Create<IConsole>(List));
 
-        private static Command ListProcessesCommand() =>
+        private static Command ProcessStatusCommand() =>
             new Command(
-                "list-processes",
+                "ps",
```

Comment:
Although it makes me a little nervous to rename commands we already shipped, lets try it and be sure to be attentive to feedback.

Reply:
Alternatively, in order to avoid confusion in the short term I could probably keep the `list-processes` command that just prints out `list-processes command has been renamed to ps. `. What do you think?

Reply:
I was thinking about less aggressive things, but I'm game to see what happens if we just rip off the bandaid : ) Unlike runtime/BCL changes, the worst outcome I forsee here is customers say it was confusing and we update it again. If we get in the habit of being very cautious then we'll probably never get the feedback that we are being more cautious than we need to be.

Reply:
We could perhaps forward `list-processes` to the new `ps` command.  I'm game for just making the change, though.